core/src/test/scala/org/apache/spark/util/CompletionIteratorSuite.scala

```diff
@@ -64,6 +64,6 @@ class CompletionIteratorSuite extends SparkFunSuite {
       }
     }
     assert(ref.refersTo(null))
-    assert(refQueue.poll() === ref)
+    assert(refQueue.remove() === ref)
```

Comment:
cc @dongjoon-hyun @yaooqinn FYI

Reply:
`remove` is a blocking method, should we add a `timeout(ms)`? like `.remove(1000)`

Reply:
also cc @srowen